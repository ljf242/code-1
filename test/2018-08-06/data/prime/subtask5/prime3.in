65184547091098 65184547191092 1000000000
