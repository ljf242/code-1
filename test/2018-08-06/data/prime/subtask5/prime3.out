65184547155669
